cmd/sentry.cpp

```diff
@@ -38,6 +38,17 @@ int sentry_main(int argc, char* argv[]) {
     cli.add_option("--sentry.api.addr", options.api_address, "GRPC API endpoint")
         ->capture_default_str()
         ->check(IPEndPointValidator(/*allow_empty=*/true));
+    cli.add_option("--port", options.port, "Network listening port")
```

Comment:
Not sure what this `--port` is for but need to be enforced a check that is a valid number in [1-65535]